tests/test_interface/test_03/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_executable(svZeroD_interface_test03 ../LPNSolverInterface/LPNSolverInterface.cpp  main.cpp)
+target_link_libraries(svZeroD_interface_test03 ${CMAKE_DL_LIBS})
```

Comment:
It's not necessary for this, but it would be nice to include these tests in our coverage report. Maybe adding the coverage code from our main CMakeLists.txt in here would be sufficient?

Reply:
Will create issue